test/runtime-puppeteer/index.ts

```diff
@@ -240,6 +227,40 @@ describe('runtime (puppeteer)', () => {
 					throw new Error('Received unexpected warnings');
 				}
 			}
+			// NOTE: Chromium may exit due to SIGSEGV, so retry in that case.
```

Comment:
it looks like the same code exists in `test/custom-elements/index.ts`. I wonder if we should have a file to share the logic?

Reply:
I moved common logic to the shared file.
Please check this commit!
https://github.com/sveltejs/svelte/pull/7076/commits/6e733b7700c08a75365482f8397221d2079ed6b3